lib/src/components/material_yes_no_buttons/material_yes_no_buttons.dart

```diff
@@ -266,8 +266,8 @@ class EscapeCancelsDirective extends BoundaryAwareKeyUpDirective
   void _onMatchingKeyUp(KeyboardEvent event) => _yesNo.no.add(true);
 }
 
-/// If attached to the yes-no buttons it will listen for enter`keyup` event
-/// and trigger [yes] action on it.
+/// If attached to yes-no buttons, it will listen for enter `keyup` events
```

Comment:
enter -> Enter? Enter/Return?

(I was trying to parse "enter" as a verb at first.)